RealmBrowser/Classes/RLMApplicationDelegate.m

```diff
@@ -94,6 +96,27 @@ - (BOOL)application:(NSApplication *)application openFile:(NSString *)filename
     return YES;
 }
 
+- (void)application:(NSApplication *)sender openFiles:(NSArray *)filenames
+{
+    if (filenames.count > kMaxNumberOfFilesAtOnce) {
+        NSString *message = [NSString stringWithFormat:@"Are you sure you wish to open all %lu Realm files?", (unsigned long)filenames.count];
+        
+        NSAlert *alert = [[NSAlert alloc] init];
+        [alert setMessageText:message];
+        [alert setInformativeText:@"Opening too many files at once may result in system instability."];
```

Comment:
Yeah I agree. This is supposed to be more of a stopgap solution than an actual fix.

In the scenario I was having for my app, I'm only ever interested in opening the default.realm file, so perhaps even some kind of prompt where you can selectively pick the .realm file you want to open would be a bit more elegant.